graphql_compiler/query_pagination/query_parameterizer.py

```diff
@@ -86,7 +86,7 @@ def _is_new_filter_stronger(
     Returns:
         whether the old filter can be removed with no change in query meaning.
     """
-    vertex_type = query_analysis.types[property_path.vertex_path].name
+    vertex_type = query_analysis._types[property_path.vertex_path].name
```

Comment:
We could make `types` public.

Reminder how it's different than the type information in the `metadata_table`: It doesn't care about revisits.